persistence-cassandra-4.0/pom.xml

```
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.stargate</groupId>
+    <artifactId>stargate</artifactId>
+    <version>0.0.11-SNAPSHOT</version>
+  </parent>
+  <groupId>io.stargate.db.cassandra</groupId>
+  <artifactId>persistence-cassandra-4.0</artifactId>
```

Do we want to keep the original version?

```suggestion
  <groupId>io.stargate.db.cassandra</groupId>
  <artifactId>persistence-cassandra-4.0</artifactId>
  <version>0.0.5-SNAPSHOT</version>
```